src/main/java/com/minelittlepony/PonyConfig.java

```diff
@@ -1,40 +1,49 @@
 package com.minelittlepony;
 
 import com.google.gson.annotations.Expose;
+import com.minelittlepony.pony.data.PonyLevel;
 import com.mumfrey.liteloader.modconfig.ConfigStrategy;
 import com.mumfrey.liteloader.modconfig.Exposable;
 import com.mumfrey.liteloader.modconfig.ExposableOptions;
 
+/**
+ * Storage contained for MineLP client settings.
+ *
+ */
 @ExposableOptions(filename = "minelittlepony", strategy = ConfigStrategy.Unversioned)
 public class PonyConfig implements Exposable {
 
-    @Expose
-    private PonyLevel ponylevel = PonyLevel.PONIES;
-    @Expose
-    public boolean sizes = true;
-    @Expose
-    public boolean snuzzles = true;
-    @Expose
-    public boolean hd = true;
-    @Expose
-    public boolean showscale = true;
-    @Expose
-    public boolean villagers = true;
-    @Expose
-    public boolean zombies = true;
-    @Expose
-    public boolean pigzombies = true;
-    @Expose
-    public boolean skeletons = true;
-    @Expose
-    public boolean illagers = true;
+    @Expose private PonyLevel ponylevel = PonyLevel.PONIES;
+    @Expose public boolean sizes = true;
+    @Expose public boolean snuzzles = true;
+    @Expose public boolean hd = true;
+    @Expose public boolean showscale = true;
+    @Expose public boolean villagers = true;
+    @Expose public boolean zombies = true;
+    @Expose public boolean pigzombies = true;
+    @Expose public boolean skeletons = true;
+    @Expose public boolean illagers = true;
 
+    /**
+     * Gets the current PonyLevel. That is the level of ponies you would like to see.
+     * @param ignorePony true to ignore whatever value the setting has.
+     */
+    public PonyLevel getPonyLevel(boolean ignorePony) {
+        return ignorePony ? PonyLevel.BOTH : getPonyLevel();
+    }
+
+    /**
+     * Actually gets the pony level value. No option to ignore reality here.
+     */
     public PonyLevel getPonyLevel() {
-        if (ponylevel == null)
-            ponylevel = PonyLevel.PONIES;
+        if (ponylevel == null) ponylevel = PonyLevel.PONIES;
```

Comment:
Keep this on separate lines

Reply:
That if statement was removed in an later commit.

```    public PonyLevel getEffectivePonyLevel(boolean ignorePony) {
        return ignorePony ? PonyLevel.BOTH : getPonyLevel();
    }```

On the matter of broken if statements: No. Either use braces of put them on one line. Two-line no braces is just plain hard to read.

Reply:
I completely derped this one, sorry. Changed to use braces.